libbeat/cmd/instance/metrics_other.go

```diff
@@ -1,8 +1,20 @@
-// +build !darwin,!linux,!windows darwin,!cgo linux,!cgo windows,!cgo
+// +build !darwin,!linux,!cgo darwin,!cgo freebsd,!cgo openbsd
 
 package instance
 
-import "github.com/elastic/beats/libbeat/logp"
+import (
+	"github.com/satori/go.uuid"
+
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+var (
```

Comment:
Could we move the ephemeralID part to a third file which gets compile on all platforms? Like this we can make sure we only have to change it in one place.